presto-base-jdbc/src/test/java/io/prestosql/plugin/jdbc/TestJdbcQueryBuilder.java

```diff
@@ -237,11 +236,22 @@ public void testNormalBuildSql()
                 .build());
 
         Connection connection = database.getConnection();
-        try (PreparedStatement preparedStatement = new QueryBuilder(jdbcClient).buildSql(SESSION, connection, TEST_TABLE, Optional.empty(), columns, tupleDomain, Optional.empty(), identity());
-                ResultSet resultSet = preparedStatement.executeQuery()) {
+        try (PreparedStatement preparedStatement = new QueryBuilder(jdbcClient).buildSql(SESSION, connection, TEST_TABLE, Optional.empty(), columns, tupleDomain, Optional.empty(), identity())) {
+            assertThat(lastQuery).isEqualTo("" +
+                    "SELECT \"col_0\" AS \"col_0\", \"col_1\" AS \"col_1\", \"col_2\" AS \"col_2\", \"col_3\" AS \"col_3\", \"col_4\" AS \"col_4\", \"col_5\" AS \"col_5\", " +
+                    "\"col_6\" AS \"col_6\", \"col_7\" AS \"col_7\", \"col_8\" AS \"col_8\", \"col_9\" AS \"col_9\", \"col_10\" AS \"col_10\", \"col_11\" AS \"col_11\" " +
+                    "FROM \"test_table\" " +
+                    "WHERE ((\"col_0\" < ?) OR (\"col_0\" >= ? AND \"col_0\" <= ?) OR (\"col_0\" > ?) OR \"col_0\" IN (?,?)) " +
+                    "AND ((\"col_1\" >= ? AND \"col_1\" <= ?) OR (\"col_1\" >= ? AND \"col_1\" <= ?) OR \"col_1\" IN (?,?,?,?)) " +
+                    "AND ((\"col_7\" >= ? AND \"col_7\" < ?) OR (\"col_7\" >= ? AND \"col_7\" < ?)) " +
+                    "AND ((\"col_8\" >= ? AND \"col_8\" < ?) OR (\"col_8\" >= ? AND \"col_8\" <= ?)) " +
+                    "AND ((\"col_9\" < ?) OR \"col_9\" IN (?,?)) " +
+                    "AND (\"col_2\" = ?)");
```

Comment:
I don't like this assertion it is very detailed and so it it will be fragile to changes

Reply:
It is required to visualize the changes and this is the only place where we test queries output to remote system.
Also, the structure of output queries changes very very rarely, so expected maint cost is low.